src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {

```

Comment:
we should parametrize an interface MachineProducer with the correct type. In our case it should be `Bulldozer`

Also, we should return list of `Bulldozers` from method `get()`